frontend/queries/courseWithEnrollment.ts

```diff
@@ -6,12 +6,25 @@ import { ENROLLMENT_FRAGMENT } from "./enrollmentFragment";
 export const COURSE_WITH_ENROLLMENT = gql`
   ${COURSE_FRAGMENT}
   ${ENROLLMENT_FRAGMENT}
-  query CourseWithEnrollment($id: Int!) {
+  query CourseWithEnrollment($id: Int!, $authId: uuid!) {
     Course_by_pk(Id: $id) {
       ...CourseFragment
       Enrollments {
         ...EnrollmentFragment
       }
+      Sessions {
+        Id
+        Finish
+        CourseId
+        Description
+        Location
+        Start
+        Title
+        Attendences(where: { Person: { AuthId: { _eq: $authId } } }) {
+          Id
+          Attending
+        }
+      }
```

Comment:
Awesome this is working 🎉 
I think I would still prefer to have the session thing in a fragment as they are a copy here right now.

Reply:
Yes, after finalising WIP there now is a fragment.